tests/test_data.py

```diff
@@ -10,7 +11,7 @@
             "points",
             id="1",
         ),
-        pytest.param("points2", id="2"),
+        pytest.param("points", id="2"),
```

Comment:
remove this altogether - the test no longer needs to be parametrized

Reply:
Done